runtime/cere-dev/src/lib.rs

```diff
@@ -909,7 +909,7 @@ parameter_types! {
 	pub const ProposalBond: Permill = Permill::from_percent(5);
 	pub const ProposalBondMinimum: Balance = 50_000 * DOLLARS;
 	pub const SpendPeriod: BlockNumber = DAYS;
-	pub const Burn: Permill = Permill::from_parts(0);
+	pub const Burn: Permill = Permill::from_parts(580);
```

Comment:
As far as I remember, it's 5.8%, which is one order of magnitude more. Could you double-check?

Reply:
It is actually `0.058%`